apollo-router/src/query_planner/bridge_query_planner.rs

```diff
@@ -819,6 +811,153 @@ impl QueryPlan {
     }
 }
 
+fn standardize_schema(mut schema: apollo_compiler::Schema) -> apollo_compiler::Schema {
+    use apollo_compiler::schema::ExtendedType;
+
+    fn standardize_value_for_comparison(value: &mut apollo_compiler::ast::Value) {
+        use apollo_compiler::ast::Value;
+        match value {
+            Value::Object(object) => {
+                for (_name, value) in object.iter_mut() {
+                    standardize_value_for_comparison(value.make_mut());
+                }
+                object.sort_by_key(|(name, _value)| name.clone());
+            }
+            Value::List(list) => {
+                for value in list {
+                    standardize_value_for_comparison(value.make_mut());
+                }
+            }
+            _ => {}
+        }
+    }
+
+    fn standardize_directive_for_comparison(directive: &mut apollo_compiler::ast::Directive) {
+        for arg in &mut directive.arguments {
```

Comment:
should arguments be sorted or are we sure they are generated in the same way on both js and rust?